src/app/modules/benefit/pages/sending/sending.component.ts

```diff
@@ -37,19 +37,26 @@ export class BenefitSendingComponent implements AfterContentInit  {
         // After view inits, begin sending the application
         this.transmissionInProcess = true;
         this.hasError = false;
-        // this.logService.log({name: 'PA - application submitting request'},"PA : Submission Request");
+        this.logService.log(
+            {
+                name: 'Supplementary Benefits - application submitting request'
+            },
+            "PA : Submission Request"
+        );
```

Comment:
Same as above, this would run every time someone uses benefit, with or without error